lib/features/auth/presentation/pages/login_form.dart

```diff
@@ -0,0 +1,239 @@
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:uome2/core/themes/app_colors.dart';
+
+class LoginForm extends StatefulWidget {
+  const LoginForm({Key? key}) : super(key: key);
+
+  @override
+  State<LoginForm> createState() => _LoginFormState();
+}
+
+class _LoginFormState extends State<LoginForm> {
+  bool _passwordVisible = true;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: MediaQuery.of(context).size.height * 0.95,
+      decoration: const BoxDecoration(
+        color: AppColors.lightGrey,
+        borderRadius: BorderRadius.only(
+          topLeft: Radius.circular(15),
```

Comment:
Check if we can use `BorderRadius.vertical` or `BorderRadius.horizontal`

Reply:
yes it is possible